pkg/filesystem/mount_unix.go

```diff
@@ -0,0 +1,130 @@
+//go:build darwin || freebsd || linux
+// +build darwin freebsd linux
+
+package filesystem
+
+import (
+	"errors"
+	"fmt"
+	"unsafe"
+
+	"github.com/buildbarn/bb-storage/pkg/util"
+
+	"golang.org/x/sys/unix"
+)
+
+const (
+	FSCONFIG_SET_FLAG        = 0
```

Comment:
These are constants from the Linux kernel, right? If so, could you please submit a PR against `golang.org/x/sys/unix`'s bindings generator to auto-extract those?